x-pack/plugins/data_visualizer/public/application/common/components/expanded_row/geo_point_content_with_map/geo_point_content_with_map.tsx

```diff
@@ -17,16 +16,17 @@ import { useDataVisualizerKibana } from '../../../../kibana_context';
 import { JOB_FIELD_TYPES } from '../../../../../../common';
 import { ES_GEO_FIELD_TYPE, LayerDescriptor } from '../../../../../../../maps/common';
 import { EmbeddedMapComponent } from '../../embedded_map';
+import { ExpandedRowPanel } from '../../stats_table/components/field_data_expanded_row/expanded_row_panel';
 
 export const GeoPointContentWithMap: FC<{
   config: FieldVisConfig;
-  indexPattern: IndexPattern | undefined;
+  indexPattern: DataView | undefined;
```

Comment:
Not sure it should go with `DataView` into 7.16?

Reply:
We should keep all visible text as 'index pattern' rather than 'data view' in this PR, and if the `DataView` type is not available in 7.x, I'd suggested leaving it as `IndexPattern` for `master` too.

Reply:
Good point! I initially thought it was ok to update since IndexPattern is marked as deprecated, but it makes sense to not to do that for 7.x. I reverted those changes here https://github.com/elastic/kibana/pull/112870/commits/c0a5bda440c1a0ec84b32bb2eb0a61bfdecbaf27